rust/noosphere-cli/README.md

```diff
@@ -5,3 +5,56 @@
 The Noosphere CLI is a reference client and pedagogical tool to demonstrate
 the principles of the Noosphere protocol and give interested users a
 no-code, low-complexity tool to synchronize content through the Noosphere.
+
+
+## Usage
+### User perspective
+```sh
+# Create an identity
+orb key create `whoami`
+
+# make a directory for your sphere.
+mkdir my-sphere
+cd my-sphere
+
+# Generate your personal sphere.
+orb sphere create --owner-key `whoami`
+
+# ..now make edits ..
+
+# see the status of files in your directory
+orb status
+
+# persist changes to the sphere
+orb save
+
+# join a gateway, after you've set this identity as the counterpart
```

Comment:
"join" means something else in this context e.g. in `orb sphere join`

Reply:
How do you mean? From the linked video, this is what was typed to join the gateway.

Reply:
i just mean the comment "join a gateway" in a pedantic-sense that maybe there's a better verb here (e.g. "register|subscribe to|set a gateway"), as we use `orb sphere join` later where "join" means synchronize another sphere (which is kinda what's going on here, admittedly)